core/src/main/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactory.java

```diff
@@ -216,6 +216,25 @@
                       .collect(toImmutableList());
     }
 
+    /**
+     * Returns the list of {@link AnnotatedHttpService} defined by {@link Path} and HTTP method annotations
+     * from the specified {@code object}, {@link ExceptionHandlerFunction}'s,
+     * {@link RequestConverterFunction}'s and {@link ResponseConverterFunction}'s.
+     */
+    public static List<AnnotatedHttpServiceElement> find(String pathPrefix, Object object,
+                                                         List<ExceptionHandlerFunction>
+                                                                 exceptionHandlerFunctions,
+                                                         List<RequestConverterFunction>
+                                                                 requestConverterFunctions,
+                                                         List<ResponseConverterFunction>
+                                                                 responseConverterFunctions) {
```

Comment:
nit: I think you could break the line right after `find(` to make this look better.